### DISCUSSION TITLE
there is no "GitHub Pages" in setting, how to publish my files

### DISCUSSION THREAD

**Discussion by ocean72** [community/community#32863] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi all!
after clicking commit new file button to resist "index.html",
click "setting" to publishing, 

there is no "GitHub Pages" in this page, when I scroll down this page.
8:08
https://www.youtube.com/watch?v=RBlv2aKKWg4&ab_channel=%E7%9B%B4%E4%B9%9F%E3%83%86%E3%83%83%E3%82%AF

what should I do next?
Thank you very much for taking your time.

**Comment by airtower-luna** [comment, marked as answer]
The UI has changed a bit since that video was made. There should be a "Pages" entry in the menu on the left of the repository settings page, click that.

![grafik](https://user-images.githubusercontent.com/5814919/189831250-f987439a-2e70-4a3b-a8f5-72dcd754a20f.png)

**Reply by ocean72** [reply]
thank u very much for your advice.
I could change "none" to "main" like picture.

how to make a link to show my files to customers

ex)
"ocean11" is my account and repository name is "hello"
and I put index.html and style.css and jpec files.
how to make a link
please tell me.
![2022-09-13_22h45_15](https://user-images.githubusercontent.com/113416688/189921501-83855845-7b15-472d-927b-b2eaa2f8217b.jpg)

**Reply by airtower-luna** [reply]
In general the format will be `https://<username>.github.io/<repository>/`, so in your example `https://ocean11.github.io/hello/`.

After a successful deployment the Pages settings page should show the URL. And if you check the log of the `deploy` job you should see the URL there, too, look for `page_url`.

**Reply by zivzachr** [reply]
where can i add themes in this settings!?

**Reply by airtower-luna** [reply]
You can't. If you use the default Jekyll build you need to [do that in the Jekyll config file](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/adding-a-theme-to-your-github-pages-site-using-jekyll), otherwise it depends on whatever other tools you are using.

**Reply by JconnectTech** [reply]
how do I launch the GitHub pages

**Reply by titanmind** [reply]
my god. this is a mess now. lol. just went through several videos, articles, and every competent LLM and nothing gets anything working through github pages anymore.

**Comment by ocean72** [comment]
and I'm uploading my files, index.html, css, img files.
like picture, jpeg files in img file and style.css in css folder
plz tell me how to upload correctry 
I dont know how to upload different class; stratum files.
(now I rewrite my index.html like "<link rel="stylesheet" href="css/style.css">"→"<link rel="stylesheet" href="style.css">
and like this, img src="img/0002.jpg"→img src="0002.jpg"
thank you very much for taking your time.
![2022-09-14_08h27_02](https://user-images.githubusercontent.com/113416688/190027711-ed0b76f3-a65b-4bb4-81a4-1ab1ef4e3fbc.jpg)

**Reply by radar-blip** [reply]
You said before your account is ocean11, did you mean ocean72?

**Reply by ocean72** [reply]
yes, sorry for making you complicated

**Comment by airtower-luna** [comment]
It's easiest if you clone the repository locally instead of uploading through the web interface. Then you can commit your files with any directory structure you want, and push to the GitHub repository.

If you're not familiar with Git yet, check the [Git Book](https://git-scm.com/book/en/v2) site. There are various translations, check on the left for a list. The first three chapters cover the most important parts.

**Comment by Goldflow** [comment]
Relevant for 2024: https://github.blog/changelog/2024-07-08-pages-legacy-worker-sunset/

**Comment by padath314** [comment]
![image](https://github.com/user-attachments/assets/3920afaf-635c-41de-bcdb-6ae3ecc4f268)
